src/common/helsinkiLogo/__tests__/HelsinkiLogo.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import i18n from '../../test/testi18nInit';
+import HelsinkiLogo from '../HelsinkiLogo';
+
+type Props = {
+  isLinkToFrontPage?: boolean;
+};
+
+const getWrapper = (props?: Props) => {
+  return shallow(<HelsinkiLogo {...props} />);
+};
+
+describe('renders correct logo based on language', () => {
+  test('finnish logo', () => {
+    const wrapper = getWrapper();
+    expect(wrapper.find('.logoFi').length).toEqual(1);
+  });
+
+  test('swedish logo', () => {
+    i18n.changeLanguage('sv');
```

Comment:
How well are these test sandboxed, I can't remember 🤔I think that once you change the language, it'll remain as the language you have switched to for the ensuing tests. For consistency, maybe it could be worth it to set the language back into the default language with an `afterEach` etc.

Reply:
Next test below this one has language set to `fi`, so it seems that language change is affecting only single test.